### ISSUE TITLE
BUG: Trying to cancel Faction Play match search at the same moment a match is found blocks the UI

### ISSUE THREAD

**Issue by TheFifthHorseman** [MechWarriorOnline/issue-tracker#162]
If the player attempts to cancel FP match search at the same moment the game finds a match, the lobby UI is completely blocked by the "Cancelling" spinner until the match begins, preventing the player from interacting with the team chat or changing drop deck. This is obviously a bug.

**Comment by tauSentry** [comment]
Administrivia...  
@TheFifthHorseman, this looks like a duplicate of #73. Can you confirm.

**Comment by tauSentry** [comment]
Hey @TheFifthHorseman, I'm gonna go ahead and close this issue and cross-link it to #73.
